cmd/fleetctl/get_test.go

```diff
@@ -565,16 +565,16 @@ spec:
     metadata: ""
```

Comment:
We should add a tests to:

- make sure we can still properly parse in `apply` a config with nanoseconds
- make sure we can parse in `apply` a config with the new humanized string format